ibverbs-sys/build.rs

```diff
@@ -10,33 +11,47 @@ fn main() {
     println!("cargo:rustc-link-lib=ibverbs");
 
     // initialize and update submodules
-    if Path::new(".git").is_dir() {
+    if Path::new("../.git").is_dir() {
         Command::new("git")
             .args(&["submodule", "update", "--init"])
             .status()
             .expect("Failed to update submodules.");
     } else {
         assert!(
-            Path::new("vendor/rdma-core").is_dir(),
+            Path::new("ibverbs-sys/vendor/rdma-core").is_dir(),
             "vendor source not included"
         );
     }
 
     // build vendor/rdma-core
-    Command::new("bash")
-        .current_dir("vendor/rdma-core/")
-        .args(&["build.sh"])
-        .status()
-        .expect("Failed to build vendor/rdma-core using build.sh");
+    let dst = cmake::Config::new("vendor/rdma-core")
+        .define("NO_MAN_PAGES", "1")
+        .build();
```

Comment:
I keep wondering whether we should even build `rdma-core` as part of building this crate. Ideally we'd _just_ generate the bindings, and just require that the user already have `libibverbs` available from _somewhere_. What do you think?

Reply:
That's a good question. Let me look around and see what other sys crates do. Vendoring is definitely nice since it means you don't need to pull in external sources. That said, making it mandatory is not great. What I can do is make it a feature flag and only used a vendored version (from a tarball or something) if the flag is enabled.